<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::HLine</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QLabel" name="Dictionary">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>400</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>E-Dictionary</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>250</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>381</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="InsertButton">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add New</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QLabel" name="input_label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Search Word</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="input_word">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="start_search">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="meaning_output">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>400</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Type a word and press Search</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="return_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="display_time">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>141</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="data_struc">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>Ternary Search Tree</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string>Select Data Structure</string>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Linked List</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Trie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ternary Search Tree</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hash Table</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vectors</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QLabel" name="insert_word_label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>14</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Add New word</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>390</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="wordToInsert">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="meaning">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="partOfSpeech">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>Noun</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Noun</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Verb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Adjective</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Adverb</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Word:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Part Of Speech:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Meaning:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
